build/dependencies.props

```diff
@@ -33,10 +33,10 @@
     <MicrosoftEntityFrameworkCoreDesignPackageVersion>3.0.0-preview6.19304.10</MicrosoftEntityFrameworkCoreDesignPackageVersion>
```

Comment:
I cant update this because this depends on netstandard2.1. @HaoK are there breaking changes after preview6 that can impact scaffolding?